native/cocos/renderer/gfx-vulkan/VKSwapchain.cpp

```diff
@@ -258,6 +258,9 @@ void CCVKSwapchain::doResize(uint32_t width, uint32_t height, SurfaceTransform /
 }
 
 bool CCVKSwapchain::checkSwapchainStatus(uint32_t width, uint32_t height) {
+    if (_gpuSwapchain->vkSurface == VK_NULL_HANDLE) { // vkSurface will set to VK_NULL_HANDLE after call doDestroySurface
```

Comment:
will set -> will be set.

Why this function will be invoked after surface is destroyed.

Reply:
called from ts